.github/workflows/ci.yaml

```
@@ -0,0 +1,22 @@
+name: CI
+
+on: [push]
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+
+        strategy:
+            matrix:
+                node-version: [8.x, 10.x, 12.x]
```

```suggestion
                node-version: [8, 10, 12, 14]
```

Removing the `.x` is just a minor preference of mine, main point is to add 14.